Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -965,6 +1151,17 @@ public SessionDetails MakeSessionHandleFromSearch(string sessionId)
 
         #region Online Session Lookup and Search
 
+        /// <summary>
+        /// Performs an online Search to look up a Session, where all attributes provided are met.
+        /// <see cref="OnFindSessionsCompleteCallback"/> is called with the results.
+        /// A handle for retriving results will be stored in <see cref="CurrentSearch"/>.
+        /// 
+        /// TODO: I feel like this should run a callback or return a Result in case of errors when trying to set up the search.
```

Comment:
Particularly for comments that contain personal statements or opinions, you should either indicate that it is a `NOTE:` to yourself or rephrase the comment such that it is more objective. As it is written, it is not really a `TODO:` item that is written for any developer to take action on.

Reply:
I'm willing to put on my executive action pants on, and restate this to "this should be done" rather than "I feel like this should".